ush/config_defaults.yaml

```diff
@@ -1461,10 +1461,43 @@ task_make_orog:
       rundir: "{{ task_make_orog.rundir }}/shave4"
   filtered_output: "{{ task_make_orog.filter_topo.rundir }}/{{ task_make_orog.filter_topo.config.filtered_orog }}"
 task_make_sfc_climo:
-  KMP_AFFINITY_MAKE_SFC_CLIMO: "scatter"
-  OMP_NUM_THREADS_MAKE_SFC_CLIMO: 1
-  OMP_STACKSIZE_MAKE_SFC_CLIMO: "1024m"
   SFC_CLIMO_DIR: '{{ workflow.EXPTDIR + "/sfc_climo" if rocoto.tasks.get("task_make_sfc_climo") else "" }}'
+  sfc_climo_gen:
+    execution:
+      batchargs:
+        export: NONE
+        nodes: 1
+        tasks_per_node: 24
+        stdout: "{{ user.HOMEdir }}/scripts"
+        walltime: 00:02:00
```

Comment:
I was snooping around in our test areas on disk and noticing that there were at least a couple of times the task succeeded in 125 seconds. Since these are all low-res tests, we might consider bumping this up to 5 mins for the default so folks aren't getting surprise failures for higher resolution experiments.

I think the 20 min default in the original code is a bit much.